src/index.css

```diff
@@ -1,88 +1,112 @@
-* {
-  box-sizing: border-box;
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
   margin: 0;
   padding: 0;
-  font-family: Arial, Helvetica, sans-serif;
-}
-
-body {
   font-family: 'Roboto', sans-serif;
 }
 
-header {
+.header_div {
   width: 100%;
-  margin-left: 2rem;
   display: flex;
   flex-direction: column;
   justify-content: center;
   height: 10vh;
+  background-color: rgb(89, 17, 17);
 }
 
 .heading {
   font-size: 2.5rem;
-  transition: color 3s;
+  font-size: 6rem;
+  font-weight: 700;
+  color: #fff;
   text-align: center;
   margin-bottom: 20px;
   margin-top: 50px;
 }
 
-.heading:hover {
-  color: #fde74cff;
-}
-
-main {
+.main {
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
   width: 100%;
+  background-image: url(image/background.jpeg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: auto;
 }
 
-.space {
-  width: 40%;
+.display_card {
+  display: flex;
+  flex-direction: row;
+  max-width: 1500px;
+  width: 100%;
+  height: 100vh;
+  gap: 2rem;
+  justify-content: space-between;
 }
 
 .displayScore {
   display: flex;
   flex-direction: column;
+  width: 55%;
+  height: 1000px;
```

Comment:
- [ ] [REQUIRED CHANGES] Please, let's improve our style here as the height of the container is way too big thereby making your design somehow not what you want for your user experience. try adjusting the height here to improve the design.

- [ ] [OPTIONAL]  when you have many scores on your board, the board container overlaps the footer, I think this is also not what you want. Having a good design is a skill you wanna have in your arsenal. please try making these few adjustments.